_posts/2019-04-17-cockpit-192.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Cockpit 192
+author: pitti
+date: 2019-04-17
+tags: cockpit linux
+slug: cockpit-192
+category: release
+summary: Cockpit with Machines and translation improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 192.
+
+### Machines: Auto-detect guest operating system
+
+When creating a new VM, the "OS Vendor" and "Operating System" fields are now
+set automatically after specifying the installation sources. This uses the
+libvirt [osinfo-detect](https://libosinfo.org/) utility.
+
+![Machines OS auto-detection](/images/machines-os-autodetect.png)
+
+### Translation cleanup
+
+The language dialog now offers all available translations instead of just a
+fixed subset, orders them alphabetically, and all languages are now shown with
+their native translation:
```

Comment:
This paragraph is a little odd, as it goes from now to not to features of now.

I'd suggest something like this:

> Instead of showing just a predetermined set of languages, the language dialog now offers all available translations, ordered alphabetically, in their native translations.